### PR TITLE
About Us style #10

### LINKED ISSUE #10: About us
#1
as a user i want to knew more about elegant website 

**Acceptance Criteria**

- [ ] display about us information from database
- [ ] Styling the page

### PR DESCRIPTION
**Pull request by walaamedhat**
- fixes #10  

- related to #10   

- make aboutus page responsive

### CODE REVIEW

File: public/aboutus.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        
+    <div><img src="./image/logo.png" alt="logo" class="logo"></div>
+    <div class="burgerButton">
+          <i class="material-icons "id="burgerButton">menu</i>
+      </div>
+    </div>
+
+    <div class="container" id="container">
+      <h1 class="h1">About Us</h1>
+      <center><div class="parag"><p class="p">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem.</p>
```
Comment (MahmoudMH) [inline public/aboutus.html]: you should don't use tag center anymore as mush said you. 
div ! why you use it ? you can put it inside !.. so we can discuss it if any problem

**Review by muhmushtaha — CHANGES_REQUESTED** [review]
- the text takes a width more than the parent in small screens
- fix the conflicts @walaamedhat

**Comment by walaamedhat** [comment]
changes done

File: public/aboutus.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  </div>
+
+  <div class="container" id="container">
+    <h1 class="h1">About Us</h1>
+    <div class="parag">
+      <p class="p">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
+        It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with
+        desktop publishing software like Aldus PageMaker including versions of Lorem.</p>
+    </div>
+    <center>
```
Comment (muhmushtaha) [inline public/aboutus.html]: center tag again?

**Comment by muhmushtaha** [comment]
pull from the master on this branch and fix the conflicts
@walaamedhat

**Review by Salam-Dalloul — DISMISSED** [review]

**Review by muhmushtaha — APPROVED** [review]

**Pull request merged**